modules/light-clients/08-wasm/types/types_test.go

```diff
@@ -34,8 +34,8 @@ import (
 )
 
 const (
-	tmClientID      = "07-tendermint-0"
-	grandpaClientID = "08-wasm-0"
+	tmClientID   = "07-tendermint-0"
+	wasmClientID = "08-wasm-0"
 )
```

Comment:
At one point, I was using grandpaClientID in my test (I'm not anymore). So I had renamed it to wasmClientID. I can revert this if requested.

Reply:
reverted

Reply:
Curious, how come you reverted? I think `wasmClientID` makes more sense, doesn't it?

Reply:
It made this PR harder to review due to large diff. I want to make a new PR for this